frontend/www/js/omegaup/components/course/CardPublic.vue

```diff
@@ -8,7 +8,14 @@
             class="card-body d-flex flex-column h-100 justify-content-between"
           >
             <div>
-              <h5 class="card-title mb-0">{{ course.name }}</h5>
+              <span v-if="loggedIn">
+                <h5 card-title mb-0>
```

Comment:
Acá faltaría envolver en el atributo `class` de esta forma:
```suggestion
                <h5 class="card-title mb-0">
```
?